calendar.js

```diff
@@ -1,3 +1,78 @@
+var displayMonth = function displayMonthView() {
+  var thElements = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'], // Days of the Week
+    numberOfWeeksInAMonth = weekCount(new Date().getYear(), new Date().getMonth()), // Gives the number of weeks of the current calendar month, not necessarily from 1st to the last date of the current month, but from the first Sunday of the calendar month view to the last date.
+    firstSundayOfMonthView = new Date(getSundayOfCurrentWeek(new Date(2018, 1, 1))).getDate(), // First day of the month in the calendar view, not necessarily the first day of the month
+    monthOfFirstSundayOfCurrentWeek = new Date(getSundayOfCurrentWeek(new Date(2018, 1, 1))).getMonth(),
+    lastDateOfPreviousMonth,
+    lastDateOfCurrentMonth,
+    dateOfMonth,
+    sundayOfCurrentWeek,
+    j,
+    k;
+  if (monthOfFirstSundayOfCurrentWeek !== new Date().getMonth()) { // Incase some dates from the previous month are displayed in the current month view
+    lastDateOfPreviousMonth = getTheLastDateInTheMonth(firstSundayOfMonthView);
+    lastDateOfCurrentMonth = getTheLastDateInTheMonth(new Date());
+  } else {
+    lastDateOfCurrentMonth = getTheLastDateInTheMonth(new Date());
+  }
+  $.each(thElements, function monthDayOfWeekHeader(i, item) {
+    jQuery('<div/>', {
+      id: item,
+      role: 'columnheader',
+      class: 'PhLhOd elYzab-cXXICe-Hjleke',
+    }).appendTo('#weekOfDayHeader');
+
+    jQuery('<span/>', {
+      class: 'wy3aMe',
+      text: item,
+    }).appendTo('#' + item);
+  });
+  for (k = 0; k < numberOfWeeksInAMonth; k += 1) {
+    dateOfMonth = k === 0 ? firstSundayOfMonthView : dateOfMonth;
+    jQuery('<div/>', {
+      id: 'row' + k,
+      role: 'row',
+      class: 'QIadxc',
```

Comment:
hey @rama1990br it's okay if the UI is completely bad or broken, but please remove these weird class names everywhere. Do this first